internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -237,7 +220,7 @@ func (r *ClusterDeploymentsReconciler) Reconcile(origCtx context.Context, req ct
 		}
 	}
 
-	if swag.StringValue(cluster.Kind) == models.ClusterKindCluster {
+	if swag.StringValue(cluster.Kind) == models.ClusterKindCluster && !clusterInstall.Spec.HoldInstallation {
```

Comment:
perhaps we should log this case, or show this in conditions to the user?
people might miss the reason as for why the installation never started

Reply:
user is the one that set the spec so i'm not sure what we should add?

Reply:
buy maybe i'm wrong, @avishayt @mhrivnak what do you think?

Reply:
True. but is there a way to show that the installation did not start because HoldInstallation = True?

Reply:
Not sure there is what to show.  It's in the Spec and like @filanov said, the user set it.

Reply:
ok

Reply:
Thanks for the feedback, i added the condition and changed the message as well. 
Let me know if you think it should be different.